source/includes/api/_dataset.md

```diff
@@ -943,28 +1184,27 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/5306fd54-df71-4e20-8b34-2f
+legend            | Legend for the dataset. If provided, it will override the original dataset values. |  Object | "legend": {"long": "123", "lat": "123", "country": ["pais"], "region": ["barrio"], "date": ["startDate", "endDate"]}} |
+applicationConfig | Group of relevant props of a widget. If provided, it will override the original dataset values. |   Array | Any Text      |
 
-These process are asynchronous and not instantaneous. Immediately when triggered, these requests will cause the dataset's `status` to be set to `pending`, meaning you will not be able to issue new overwrite, concatenate or append requests. Once the request has been fully processed, the status will be automatically set to `saved`. Depending on factors like API load or the size of the data being uploaded, this may take from a few minutes to a few hours to occur. The API does not issue any notification when the asynchronous operation is finished.
+The cloned dataset has the same attributes as the source one, except for the following ones:
 
-In order to perform these operation, the following conditions must be met:
+* `name`: The current timestamp is added to the original dataset's name.
+* `published`: will be false unless specified in the request body.
 
-- the dataset's `overwrite` property must be set to `true`.
-- the dataset's `status` property must be set to `saved`.
-- the user must be logged in and match one of the following:
-  - have role `ADMIN` and belong to the same application as the dataset
-  - have role `MANAGER` and be the dataset's owner (through the `userId` field of the dataset)
+The field `clonedHost` will contain the host information about the original dataset.
 
-While they ultimately achieve a very similar end result, concatenate and append rely on different internal processes, each with its own characteristics.
+*Note: Cloning a dataset requires authentication, and you will only be able to clone datasets from apps associated to your user.*
 
-- The concatenate process relies on a slower approach to ensure the operation is atomic. Until the operation is completed, you will see the dataset data as it was before the concatenate operation was triggered. If the operation fails, the old version of the data is kept accessible as it was before the concatenation process was started. There's a known issue where concatenating new data to already existing large datasets may result in failure.
-- The append operation relies on a faster process that does not offer atomicity. The new data is simply added to the current dataset, and any queries done while this process is taking place may produce results based on the preexisting data mixed with parts of the newly added values. Should the operation fail halfway, your dataset may contain all the preexisting records as well as part (but not all) of the newly added ones.
+## Concatenate and Append Data
```

Comment:
Does this only work for document based datasets? Can I mix formats (like append JSON to a CSV dataset)?